### ISSUE TITLE
Groundnet with 'General Aviation' parking only is rejected

### ISSUE THREAD

**Issue by sfrsfrsfr** [Portree-Kid/flightgear-airports#135]
Only groundnets with at least 1 gate parking spot can be uploaded. 
But many smaller airports only service general aviation. To allow players to spawn there a proper groundet is required.
I'd suggest to e.g. add a warning to the check results like 'Warning: no 'commercial aviation'/gate parkings defined.'

![Screenshot_20210529_123632](https://user-images.githubusercontent.com/68220036/120067270-e4142280-c07a-11eb-9593-a75553fc055e.png)

**Comment by BecOzIcan** [comment]
Confirming we should be performing two separate tests : 

Groundnet Valididty : A groundnet needs at least one parking position, whatever its type ie either
- ga (general aviation)
- cargo (cargo/freighter)
- gate (commercial passenger traffic)
- mil-fighter (military fighter)
- mil-cargo (military transport)

Groundnet Adequacy : A groundnet should have enough gates to accommodate the flights scheduled in FGData Traffic files for type 
- cargo (cargo/freighter)
- gate (commercial passenger traffic)

A failed Validity Test should prevent a file upload 
A failed Adequacy Test should only trigger a warning (to the contributor and the PR reviewer) but not block the submission

ga and military flights are, in essence, not following any published recurring schedule hence are not distributed through FGData and so should not be used by FGA to assess the adequacy of a groundnet

**Comment by Portree-Kid** [comment]
Yes that is on the list for the next checker iteration. I’ll move it to the checker.
